cartographer_rviz/cartographer_rviz/drawable_submap.cc

```diff
@@ -25,11 +25,16 @@
 #include "Eigen/Geometry"
 #include "OgreGpuProgramParams.h"
 #include "OgreImage.h"
+#include "cartographer/common/make_unique.h"
 #include "cartographer/common/port.h"
 #include "cartographer_ros_msgs/SubmapQuery.h"
 #include "eigen_conversions/eigen_msg.h"
 #include "ros/ros.h"
 
+#if (QT_VERSION < QT_VERSION_CHECK(5, 0, 0))
```

Comment:
Can you not use that directly? I mean, why do you need to redefine their workaround here?

Could you link where you found it in the code?